src/indicators/drought/chirps_rainfallobservations.py

```diff
@@ -189,88 +195,153 @@ def get_filepath_seasonal_lowertercile_raster(
         )
     )
 
-    chirps_seasonal_tercile_bounds_country_filepath = (
+    return chirps_seasonal_lowertercile_country_filepath
+
+
+def get_filepath_seasonal_tercile_raster(country_iso3: str, config: Config):
+    chirps_country_dir = (
+        Path(config.DATA_DIR)
+        / config.PUBLIC_DIR
+        / config.PROCESSED_DIR
+        / country_iso3
+        / config.CHIRPS_DIR
+    )
+
+    chirps_seasonal_country_dir = (
+        chirps_country_dir / config.CHIRPS_SEASONAL_DIR
+    )
+
+    chirps_seasonal_tercile_filepath = (
         chirps_seasonal_country_dir
-        / config.CHIRPS_SEASONAL_TERCILE_BOUNDS_FILENAME.format(
-            country_iso3=country_iso3
-        )
+        / _SEASONAL_TERCILE_FILENAME.format(country_iso3=country_iso3)
+    )
+
+    chirps_seasonal_tercile_bounds_filepath = (
+        chirps_seasonal_country_dir
+        / _SEASONAL_TERCILE_BOUNDS_FILENAME.format(country_iso3=country_iso3)
     )
 
     return (
-        chirps_seasonal_lowertercile_country_filepath,
-        chirps_seasonal_tercile_bounds_country_filepath,
+        chirps_seasonal_tercile_filepath,
+        chirps_seasonal_tercile_bounds_filepath,
     )
 
 
-def compute_seasonal_lowertercile_raster(
+def compute_seasonal_tercile_raster(
     config,
     country_iso3: str,
     use_cache: bool = True,
 ):
     # number of months that is considered a season
     seas_len = 3
+    end_month_season_mapping = {
+        1: "NDJ",
+        2: "DJF",
+        3: "JFM",
+        4: "FMA",
+        5: "MAM",
+        6: "AMJ",
+        7: "MJJ",
+        8: "JJA",
+        9: "JAS",
+        10: "ASO",
+        11: "SON",
+        12: "OND",
+    }
```

Comment:
This may be overkill, but I can't resist automating these things ;)
```python
from calendar import month_name
end_month_season_mapping = { i:
month_name[(i - 2) % 12][0]+ month_name[(i -1) % 12][0] + month_name[i] for i in range(1, 13)
}

Reply:
Haha I like it. Adjusted the formula a bit cause the `month_name` is 1-indexed. 
Anyway then realized, it would be even nicer to include the `seas_len` as parameter to it as well. I think the below makes sense but don't see how to do it in a list comprehension. Do you? 

``` python
end_month_season_mapping={}
for i in range(1,13):
    season_str=""
    for l in range(seas_len,0,-1):
        season_str+=month_name[(i - l) % 12+1][0]
    end_month_season_mapping[i]=season_str

Reply:
I think something like this works: 
```python
{m: ''.join([month_name[(m - i) % 12 + 1][0] for i in range(3)]) for m in range(1, 13)}
```

Reply:
Love it!